endpoints/v2/test/test_v2auth.py

```diff
@@ -217,3 +217,55 @@ def test_generate_registry_jwt(
 
     assert decoded["access"] == expected_access
     assert len(decoded["context"][CLAIM_TUF_ROOTS]) == len(expected_scopes)
+
+
+@pytest.mark.parametrize(
+    "scope, username, password, expected_code, expected_scopes",
+    [
+        (
+            "repository:devtable/newrepo:pull,push,*",
+            "devtable",
+            "password",
+            200,
+            ["devtable/newrepo:push,pull,*"],
+        ),
+    ],
+)
+def test_generate_registry_jwt_private(
+    scope, username, password, expected_code, expected_scopes, app, client
+):
+    try:
+        original_app.config["REPOSITORY_CREATION_PRIVATE"] = True
+        test_generate_registry_jwt(
```

Comment:
Instead of calling another test, you should parameterize it so that it tests for your new cases.